uint8 detail
---
uint8 status